src/routes/docs/quick-starts/go/+page.markdoc

```diff
@@ -0,0 +1,290 @@
+		fmt.Println(err)
+		return
+	}
+
+	attribute2, err := appwriteDatabases.CreateStringAttribute(
+		todoDatabase.Id,
+		todoCollection.Id,
+		"description",
+		255,
+		false,
+	)
+
+	fmt.Println(attribute2)
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	attribute3, err := appwriteDatabases.CreateBooleanAttribute(
+		todoDatabase.Id,
+		todoCollection.Id,
+		"isComplete",
+		true,
+	)
+
+	fmt.Println(attribute3)
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	seedDatabase(appwriteDatabases, todoDatabase, todoCollection)
+}
+```
+
+{% /section %}
+{% section #step-6 step=6 title="Add documents" %}
+Create a function to add some mock data to your new collection.
+
+```go
+func seedDatabase(appwriteDatabases *databases.Databases, toDoDatabase *models.Database, toDoCollection *models.Collection) {
+
+	testTodo1 := map[string]interface{}{
+		"title":       "Buy apples",
+		"description": "At least 2KGs",
+		"isComplete":  true,
+	}
+
+	testTodo2 := map[string]interface{}{
+		"title":      "Wash the apples",
+		"isComplete": true,
+	}
+
+	testTodo3 := map[string]interface{}{
+		"title":       "Cut the apples",
+		"description": "Don't forget to pack them in a box",
+		"isComplete":  false,
+	}
+
+	document1, err := appwriteDatabases.CreateDocument(
+		toDoDatabase.Id,
+		toDoCollection.Id,
+		id.Unique(),
+		testTodo1,
+	)
+
+	fmt.Println(document1)
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	document2, err := appwriteDatabases.CreateDocument(
+		toDoDatabase.Id,
+		toDoCollection.Id,
+		"unique()",
+		testTodo2,
+	)
+
+	fmt.Println(document2)
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	document3, err := appwriteDatabases.CreateDocument(
+		toDoDatabase.Id,
+		toDoCollection.Id,
+		id.Unique(),
+		testTodo3,
+	)
+
+	fmt.Println(document3)
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	getTodos(appwriteDatabases, toDoDatabase, toDoCollection)
+}
+```
+
+
+Make sure to update `main()` with the todo Collection function you created. Your `main()` function should look something like this:
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/appwrite/sdk-for-go/appwrite"
+	"github.com/appwrite/sdk-for-go/client"
+	"github.com/appwrite/sdk-for-go/databases"
+	"github.com/appwrite/sdk-for-go/id"
+	"github.com/appwrite/sdk-for-go/models"
+)
+
+func main() {
+
+	client := appwrite.NewClient(
+		appwrite.WithProject("<PROJECT_ID>"),
+		appwrite.WithKey("<API_KEY>"),
+	)
+
+	todoCollection(client)
+}
+```
+
+{% /section %}
+{% section #step-7 step=7 title="Retrieve documents" %}
+
+Create a function to retrieve the mock todo data.
+
+```go
+func getTodos(appwriteDatabases *databases.Databases, toDoDatabase *models.Database, toDoCollection *models.Collection) {
+	todos, err := appwriteDatabases.ListDocuments(
+		toDoDatabase.Id,
+		toDoCollection.Id,
+		appwriteDatabases.WithListDocumentsQueries([]string{}),
+	)
+
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	fmt.Println(todos)
```

Comment:
Let's try to print the todos in a more comprehensible way. Look at the other quick starts.